libc/src/time/ctime_s.h

```diff
@@ -0,0 +1,23 @@
+//===-- Implementation header of ctime_s ------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_LIBC_SRC_TIME_CTIME_S_H
+#define LLVM_LIBC_SRC_TIME_CTIME_S_H
+
+#include "hdr/types/rsize_t.h"
+#include "hdr/types/time_t.h"
+#include "src/__support/macros/config.h"
+#include <stddef.h>
```

Comment:
What are you using from stddef.h?  You should be using proxy headers (hdr/) here.

Reply:
Removed it.